js/index.js

```diff
@@ -0,0 +1,26 @@
+const footer = document.createElement('footer');
+const body = document.querySelector('body');
+body.appendChild(footer);
+
+const today = new Date();
+const thisYear = today.getFullYear();
+const copyright = document.createElement('p');
+copyright.innerHTML = `<span>Marilyn \u00A9 ${thisYear}</span>`;
+footer.appendChild(copyright);
+
+const skillsList = ['C++', 'Revit', 'AutoCAD', 'Microsoft Office'];
+const skillsSection = document.getElementById('Skills');
+const skillUL = skillsSection.querySelector('ul');
+
+for (skill of skillsList){
```

Comment:
You forgot to declare skill.

```suggestion
for (let skill of skillsList){
```